cs-google-cloud/src/main/scala/io/cloudslang/content/gcloud/actions/compute/instances/InstancesDelete.scala

```diff
@@ -24,8 +24,35 @@ import scala.collection.JavaConversions._
   * Created by sandorr
   * 2/27/2017.
   */
-class InstanceDelete {
+class InstancesDelete {
 
+  /**
+    * This operation can be used to delete an Instance resource. The operation returns a ZoneOperation resource as a
+    * JSON object, that can be used to retrieve the status and progress of the ZoneOperation, using the
+    * ZoneOperationsGet operation.
+    *
+    * @param projectId        Google Cloud project name.
+    *                         Example: "example-project-a"
+    * @param zone             The name of the zone in which the instance lives.
+    *                         Examples: "us-central1-a", "us-central1-b", "us-central1-c"
+    * @param instanceName     Name of the Instance resource to delete.
+    *                         Example: "operation-1234"
+    * @param accessToken      The access token returned by the GetAccessToken operation, with at least one of the
+    *                         following scopes: "https://www.googleapis.com/auth/compute",
```

Comment:
it doesn't need both of them ... it's enough:  "https://www.googleapis.com/auth/compute"